src/modules/corp-type-module/corp-type-module.ts

```diff
@@ -17,6 +17,7 @@ export enum CorpTypeCd {
   CEMETARY = 'CEM',
   COOP = 'CP',
   BC_CORPORATION = 'CR', // SPECIAL NAMEREQUEST-ONLY ENTITY TYPE
+  BC_UNLIMITED = 'UL', // SPECIAL NAMEREQUEST-ONLY ENTITY TYPE
```

Comment:
These are sorted by code. Please move it down accordingly 😛

Reply:
Sorry! Fixed 👍